lib/ansible/modules/notification/matrix.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/python
+# coding: utf-8
+
+# (c) 2018, Jan Christian Grünhage <jan.christian@gruenhage.xyz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author: "Jan Christian Grünhage (@jcgruenhage)"
+requirements: [ matrix-client ]
+module: matrix
+short_description: Send notifications to matrix
+description:
+    - This module sends html formatted notifications to matrix rooms.
+version_added: "2.8"
+options:
+    msg_plain:
+        description:
+            - Plain text form of the message to send to matrix, usually markdown
+        required: true
+    msg_html:
+        description:
+            - HTML form of the message to send to matrix
+        required: true
+    room_id:
+        description:
+            - ID of the room to send the notification to
+        required: true
+    hs_url:
+        description:
+            - URL of the homeserver, where the CS-API is reachable
+        required: true
+    token:
+        description:
+            - Authentication token for the API call. If provided, user_id and password are not required
+    user_id:
+        description:
+            - The user id of the user
+    password:
+        description:
+            - The password to log in with
+notes:
+    - Requires matrix-client on the executing host.
```

Comment:
add 'python library' , then the 'pip install' becomes redundant, some people cannot use pip to install libraries

Reply:
This was adapted from pushbullet.py, but you're concern is definitely valid, will change it.